app/scss/questionPage.scss

```diff
@@ -1,162 +1,270 @@
 @import "imports/variables";
 @import "imports/modal";
 
-.container {
-    text-align: center;
+body {
+  font-family: asap, sans-serif;
```

Comment:
font-family already set in styles.scss so you can remove it here

Reply:
styles.css isn't linked to file and linking it messes up styling so probably best to leave original infrastructure.